src/astra_driver.cpp

```diff
@@ -188,6 +189,7 @@ AstraDriver::AstraDriver(ros::NodeHandle& n, ros::NodeHandle& pnh) :
   depth_action_server_->start();
 
   diagnostics_timer_ = pnh_.createTimer(ros::Duration(1), &AstraDriver::diagnosticsTimerCallback, this);
+  last_depth_frame_timestamp_ = ros::Time::now();
```

Comment:
maybe 0? now() could cause minor hickups and race conditions when the driver starts